ui/src/app/guards/disabled-guard.service.ts

```diff
@@ -0,0 +1,29 @@
+import {Injectable} from '@angular/core';
+import {ActivatedRouteSnapshot, CanActivate, CanActivateChild, Router, RouterStateSnapshot} from '@angular/router';
+
+import {SignInService} from 'app/services/sign-in.service';
+import {profileApi} from 'app/services/swagger-fetch-clients';
+import {convertAPIError} from 'app/utils/errors';
+import {ErrorCode} from 'generated/fetch';
+
+@Injectable()
+export class DisabledGuard implements CanActivate, CanActivateChild {
+  constructor(private signInService: SignInService, private router: Router) {}
+
+  async canActivate(route: ActivatedRouteSnapshot, state: RouterStateSnapshot): Promise<boolean> {
```

Comment:
nit: I don't think you're using any async syntax, so you can drop the 'async' here and below.

I personally think the async syntax is nicer than nested, but I don't think we have a style requirement to use that (yet...)

Reply:
Want to bring it up at the eng sync? Then if everyone agrees, we can add it to our style guide markdown.